planemo/galaxy/config.py

```diff
@@ -941,7 +941,7 @@ def startup_command(self, ctx, **kwds):
         if parse_version(kwds.get('galaxy_python_version') or DEFAULT_PYTHON_VERSION) >= parse_version('3'):
             # We need to start under gunicorn
             self.env['APP_WEBSERVER'] = 'gunicorn'
-            self.env['GUNICORN_CMD_ARGS'] = "--bind={host}:{port} --name={server_name}".format(
+            self.env['GUNICORN_CMD_ARGS'] = "--capture-output --bind={host}:{port} --name={server_name}".format(
```

Comment:
gunicorn support was mostly because paster had some bugs on python 3, but that's been fixed. If it is giving you headaches we can also drop gunicorn support.